utils/utils.go

```diff
@@ -26,6 +26,15 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
+const (
+	defaultNamespace = "default"
+	DotDelimiter     = "."
+
+	// Workflow
+	WorkflowNameLabelKey = "workflow"
+	ActivityNameLabelKey = "activity"
```

Comment:
I think it's probably better to define the workflow and activity const values in the wfengine package since they're really only applicable in the context of the workflow engine.